metricsx/middleware.go

```diff
@@ -58,7 +57,7 @@ func Hash(value string) string {
 	hash := sha256.New()
 	_, err := hash.Write([]byte(value))
 	if err != nil {
-		panic(fmt.Sprintf("unable to hash value"))
+		panic("unable to hash value")
```

Comment:
Since this was here before, it's probably not an issue, but panicking in a (open source?) metrics component seems pretty dangerous.

It seems this is called on startup of the services, potentially leading to crash loops? Could we maybe improve this?

Do you have an opinion on it?